src/binding.cc

```diff
@@ -335,6 +335,20 @@ void v8__HandleScope__CONSTRUCT(uninit_t<v8::HandleScope>* buf,
 
 void v8__HandleScope__DESTRUCT(v8::HandleScope* self) { self->~HandleScope(); }
 
+void v8__Locker__CONSTRUCT(uninit_t<v8::Locker>* buf, v8::Isolate* isolate) {
+  construct_in_place<v8::Locker>(buf, isolate);
+}
+
+void v8__Locker__DESTRUCT(v8::Locker* self) { self->~Locker(); }
+bool v8__Locker__Is_Active(v8::Locker* self) { return self->IsActive(); }
+bool v8__Locker__Is_Locked(v8::Locker* self, v8::Isolate* isolate) { return self->IsLocked(isolate); }
```

Comment:
These two methods have been added for debugging the locker. I plan to remove before this PR is merged.